### ISSUE TITLE
[📍] 'ReadValueError: The test didn't produce chartjson-output.json.'

### ISSUE THREAD

**Issue by alculquicondor** [catapult-project/catapult#4548]
After #4538 being solved, I'm running into a new issue. This time I'm not sure if it's something missing in chromium side.

```
Traceback (most recent call last):
  File "/base/data/home/apps/s~chromeperf/pinpoint:dev-dtu-a490e62d.411027326553883279/dashboard/pinpoint/models/quest/execution.py", line 95, in Poll
    self._Poll()
  File "/base/data/home/apps/s~chromeperf/pinpoint:dev-dtu-a490e62d.411027326553883279/dashboard/pinpoint/models/quest/read_value.py", line 104, in _Poll
    self._isolate_server, self._isolate_hash, self._results_filename)
  File "/base/data/home/apps/s~chromeperf/pinpoint:dev-dtu-a490e62d.411027326553883279/dashboard/pinpoint/models/quest/read_value.py", line 269, in _RetrieveOutputJson
    raise ReadValueError("The test didn't produce %s." % filename)
ReadValueError: The test didn't produce chartjson-output.json.
```

https://pinpoint-dot-chromeperf.appspot.com/job/1791bbfaa40000

**Comment by simonhatch** [comment]
@dave-2

**Comment by alculquicondor** [comment]
It seems that i'm still getting the same error https://pinpoint-dot-chromeperf.appspot.com/job/16cf7b72a40000

Although, the majority of the tasks failed for a different reason.

**Comment by dave-2** [comment]
I've just now deployed a new version to ensure that the fix is in. Please re-open if you still see this with new jobs!

**Comment by dave-2** [comment]
The other failures look like WebView crashed during the test.
